experiments/EXAMPLE_gpt_raw.yml

```diff
@@ -0,0 +1,144 @@
+name: CHANGEME_your_experiment_name
```

Comment:
could you change your script to run on a different yml filename, and rename this file back to `EXAMPLE_gpt.yml` instead? The colab notebook is reliant on the `EXAMPLE_gpt.yml` file in a lot of places